packages/service-integration/src/service_integration/versioning.py

```diff
@@ -67,9 +64,9 @@ class Config:
 
 
 class ServiceVersionInfo(BaseModel):
-    version: VersionStr
-    integration_version: VersionStr = Field(
-        ..., description="osparc internal integration version"
+    version: SemanticVersionStr
+    integration_version: SemanticVersionStr = Field(
+        INTEGRATION_API_VERSION, description="osparc internal integration version"
```

Comment:
Why add a default here since it previously had none?

Reply:
you have a valid point.